util/cleanup.h

```diff
@@ -25,6 +27,12 @@ static inline void freep(void *p)
 }
 #define _cleanup_free_ __cleanup__(freep)
 
+static inline void free_huge(struct nvme_mem_huge *mh)
+{
+	nvme_free_huge(mh);
+}
+#define _cleanup_huge_ __cleanup__(free_huge)
+
 static inline void close_file(int *f)
 {
 	if (*f >= 0)
```

Comment:
Is this not necessary to check if `*f` is not STDIN_FILENO and not STDOUT_FILENO also?

Reply:
Added the filter.

Reply:
Thank you.